packages/agreement-process/src/resources/templates/documents/agreementContractTemplate.html

```diff
@@ -125,32 +125,20 @@ <h2>Registrazione</h2>
       <div>
         In data <strong>{{submissionDate}}</strong> alle ore
         <strong>{{submissionTime}}</strong>, l’Infrastruttura ha trasmesso
-        all’erogatore la richiesta di fruizione.
+        {{#if producerDelegationId}} al delegante all’erogazione{{else}} all’erogatore{{/if}} la richiesta di fruizione.
       </div>
-      {{#if producerDelegationId}}
       <div>
         In data <strong>{{activationDate}}</strong> alle ore
         <strong>{{activationTime}}</strong>, l’infrastruttura ha ricevuto
         la dichiarazione di accettazione della richiesta di fruizione
-        (di seguito “dichiarazione di accettazione”) inviata dal delegato all’erogazione,
+        (di seguito “dichiarazione di accettazione”) inviata {{#if producerDelegationId}}dal delegato all’erogazione,{{else}}inviata dall’erogatore,{{/if}}
```

Comment:
"inviata" is duplicated

```suggestion
        (di seguito “dichiarazione di accettazione”) inviata {{#if producerDelegationId}}dal delegato all’erogazione,{{else}}dall’erogatore,{{/if}}
```

Reply:
done with [1e70bca](https://github.com/pagopa/interop-be-monorepo/pull/1243/commits/1e70bca9ad05d8f2bf848b8d35fd214aa39f7dad)